packages/loader/container-loader/src/serializedStateManager.ts

```diff
@@ -91,21 +92,38 @@ export type ISerializedStateManagerDocumentStorageService = Pick<
 
 export class SerializedStateManager {
 	private readonly processedOps: ISequencedDocumentMessage[] = [];
-	private snapshot: ISnapshotInfo | undefined;
 	private readonly mc: MonitoringContext;
+	private snapshot: ISnapshotInfo | undefined;
 	private latestSnapshot: ISnapshotInfo | undefined;
 	private refreshSnapshot: Promise<void> | undefined;
+	private snapshotFetchedTime: number | undefined;
```

Comment:
can we just add the time to the ISnapshotInfo rather then needing new members to track it